src/main/java/org/opensearch/flowframework/workflow/DeleteConnectorStep.java

```diff
@@ -69,8 +69,9 @@ public void onResponse(DeleteResponse deleteResponse) {
 
             @Override
             public void onFailure(Exception e) {
-                logger.error("Failed to delete connector");
-                deleteConnectorFuture.onFailure(new FlowFrameworkException(e.getMessage(), ExceptionsHelper.status(e)));
+                String errorMessage = "Failed to delete connector";
```

Comment:
should we add connector ID here? similar for other deletion steps like agent ID and etc.